device-catalog.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="ru">
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheey" href="device-catalog.css">
+        <title>Интернет магазин Device. Каталог товаров</title>
+    </head>
+    <body>
+        <header>
+            <div>
+                <a class="nav_link" href=".//index.html"><img src=".//img/logo-black.png" alt="Logo device"></a>
+                <div>
+                    <img src=".//img/search.svg">
+                    <input type="text" name="search" placeholder="Поиск по сайту">
+                    <input type="button" value="Найти">
+                </div>
+                <ul>
+                    <li><a href="#">username</a></li> 
+                    <!-- Тут должна быть какая-то переменная для username -->
+                    <li><a href="#">Выйти</a></li>
+                    <li><a href="#">Сравнить</a></li>
+                    <li><a href="#">Корзина</a></li>
+                </ul>                
+            </div>
+            <nav class="nav">                                                
+                <ul class="nav_list">
+                    <li class="#">
+                        <div class="catalog_div">
+                        <a href="#">Каталог товаров</a>
+                        <img src=".//img/plus.svg">
+                        </div>
+                    </li>
+                    <li class="#"><a href="#">Доставка</a></li>
+                    <li class="#"><a href="#">Гарантия</a></li>
+                    <li class="#"><a href="#">Контакты</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <h2>Каталог товаров. Моноподы для селфи</h2>
+            <div class="catalog_menu">
+                <ul>
+                    <li><a href="#">Виртуальная реальность</a></li>
+                    <li><a href="#">Моноподы для селфи</a></li>
+                    <li><a href="#">Экшн-камеры</a></li>                    
+                </ul>
+                <ul>
+                    <li><a href="#">Фитнес-браслеты</a></li>
+                    <li><a href="#">Умные часы</a></li>                    
+                </ul> 
+                <ul>
+                    <li><a href="#">Квадрокоптеры</a></li>
+                </ul>                
+            </div>
+            <ul>
+                <li><a href=".//index.html">Главная</a></li>
+                <li><img src="./img/nav-arrow.svg" alt="стрелка"></li>
+                <li><a href="№">Каталог товаров</a></li>
+                <li><img src="./img/nav-arrow.svg" alt="стрелка"></li>
+                <li><a href="/device-catalog.html">Моноподы для селфи</a></li>
+            </ul>
+            <div class="sorting">
+                <p>Сортировка:</p>
+                <ul class="sorting_list">
+                    <li><button type="button">По цене</button></li>
```

Comment:
these are links. clicking on them user sees change of the url. and this sortment can be done by change of the url without click on this link